Could not get unknown property 'source' for generate-proto-generateDebugProto[Bug]

**Describe the bug**
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':barcode_scan2'.
> Could not get unknown property 'source' for generate-proto-generateDebugProto of type org.gradle.api.internal.file.DefaultSourceDirectorySet.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 8s
Error: Gradle task assembleDebug failed with exit code 1

**To Reproduce**
Steps to reproduce the behavior:
just trying to build my app on android emulator

**Expected behavior**
the build should run 


**Logs**


**Environment (please complete the following information):**
 - [ ] Emulator
 - Version of `barcode_scan` 4.3.0
 - Output of `flutter --version`: 
 Flutter 3.22.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 761747bfc5 (13 days ago) • 2024-06-05 22:15:13 +0200
Engine • revision edd8546116
Tools • Dart 3.4.3 • DevTools 2.34.3
```
Flutter ...
```

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -3,7 +3,7 @@ version '1.0-SNAPSHOT'
 
 buildscript {
     ext.kotlin_version = '1.5.20'
-    ext.protobuf_version = '0.8.18'
+    ext.protobuf_version = '0.9.1'
     ext.protocVersion = '3.11.0'
     ext.javaliteVersion = '3.11.0'
     repositories {

